package.json

```diff
@@ -22,15 +22,17 @@
     "tdd": "node scripts/tdd.js",
     "test": "SERVICES=* yarn services && mocha --exit --expose-gc 'packages/dd-trace/test/setup/node.js' 'packages/*/test/**/*.spec.js'",
     "test:core": "mocha --exit --expose-gc --file packages/dd-trace/test/setup/core.js \"packages/dd-trace/test/**/*.spec.js\"",
-    "test:core:ci": "nyc --include \"packages/dd-trace/src/**/*.js\" -- npm run test:core",
+    "test:core:ci": "nyc --include \"packages/dd-trace/src/**/*.js\" -- npm run test:core -- --reporter=mocha-junit-reporter",
```

Comment:
Does this replace any existing reporters?

Reply:
yeah it does, good catch. It doesn't seem to be possible to have multiple reporters, so I'll run it again for the JUnit XML upload (it runs fast)